plugins/modules/azure_rm_adgroup_info.py

```diff
@@ -0,0 +1,331 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2021 Cole Neubauer, (@coleneubauer)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: azure_rm_adgroup_info
+version_added: "1.4.0"
+short_description: Get Azure Active Directory group info
+description:
+    - Get Azure Active Directory group info.
+options:
+    tenant:
+        description:
+            - The tenant ID.
+        type: str
+        required: True
+    object_id:
+        description:
+            - The object id for the ad group.
+            - returns the group which has this object ID.
+        type: str
+    attribute_name:
+        description:
+            - The name of an attribute that you want to match to attribute_value.
+            - If attribute_name is not a collection type it will return groups where attribute_name is equal to attribute_value.
+            - If attribute_name is a collection type it will return groups where attribute_value is in attribute_name.
+        type: str
+    attribute_value:
+        description:
+            - The value to match attribute_name to.
+            - If attribute_name is not a collection type it will return groups where attribute_name is equal to attribute_value.
+            - If attribute_name is a collection type it will groups users where attribute_value is in attribute_name.
+        type: str
+    odata_filter:
+        description:
+            - returns groups based on the the OData filter passed into this parameter.
+        type: str
+    check_membership:
+        description:
+            - The object ID of the contact, group, user, or service principal to check for membership against returned groups.
+        type: str
+    return_owners:
+        description:
+            - Indicate whether the owners of a group should be returned with the returned groups.
+        default: False
+        type: bool
+    return_group_members:
+        description:
+            - Indicate whether the members of a group should be returned with the returned groups.
+        default: False
+        type: bool
+    return_member_groups:
+        description:
+            - Indicate whether the groups in which a groups is a member should be returned with the returned groups.
+        default: False
+        type: bool
+    all:
+        description:
+            - If True, will return all groups in tenant.
+            - If False will return no users.
+            - It is recommended that you instead identify a subset of groups and use filter
+        default: False
+        type: bool
+    log_path:
```

Comment:
log_path and log_mode has been added in parent class( azure.py and azure_rm_common.py), Please delete log_path and log_mode! same as below. Thank you very much!